test/OpenTelemetry.Extensions.Enrichment.Tests/OpenTelemetryEnrichmentProviderBuilderExtensions.cs

```diff
@@ -41,6 +41,7 @@ public void AddTraceEnricherOfTRegistersEnricher()
         using var source = new ActivitySource(SourceName);
         using (var activity = source.StartActivity(SourceName))
         {
+            Assert.NotNull(activity);
```

Comment:
We don't really need these `Assert.NotNull(activity);` statements. The unit tests would fail anyway if the activity ends up being `null`.

Reply:
I can use either use `!` as null-forgiving operator or add this assertion. Assertion looks better IMO.